src/object.h

```diff
@@ -115,7 +115,7 @@ int object_load_chunk(int groupIndex, rct_object_entry *entry, int* chunk_size);
 void object_unload_chunk(rct_object_entry *entry);
 int object_get_scenario_text(rct_object_entry *entry);
 void object_free_scenario_text();
-int object_get_length(rct_object_entry *entry);
+uintptr_t object_get_length(rct_object_entry **entryCache, size_t index);
```

Comment:
My second suggestion was to leave it and make a new function for your particular purpose.